docs/creating-service-bindings/binding-options.md

```diff
@@ -6,9 +6,14 @@ sidebar_position: 2
 
 ## Transform binding names before projection into application
 
-Binding names declared through annotations or CSV descriptors are processed before injected into the application according to the following strategy
- - names are upper-cased
- - service resource kind is upper-cased and prepended to the name
+This feature is only available for `ServiceBinding` in the
```

Comment:
maybe we could use a warning visual?  not sure how to do that with docusaurus :)